### ISSUE TITLE
gulp-jasmine runs the tests twice when Jasmine version is 2.5.2

### ISSUE THREAD

**Issue by benchristel** [sindresorhus/gulp-jasmine#86]
Hello!

It seems that tests are being run twice when `gulp-jasmine` is paired with the latest version of `jasmine`. This doesn't happen when using the `jasmine` executable by itself. The workaround is to downgrade `jasmine` to 2.4.1.

### Gulpfile

```js
var gulp    = require('gulp')
var jasmine = require('gulp-jasmine')

gulp.task('test', function() {
  return gulp.src(['test-test.js'])
    .pipe(jasmine())
})
```

### Test file

```js
// test-test.js
describe('gulp jasmine', function() {
  it('runs each test only once', function() {
    expect('foo').toEqual('fasdf')
  })
})
```

### Output of `gulp test`

```
FF

Failures:
1) gulp jasmine runs each test only once
  Message:
    Expected 'foo' to equal 'fasdf'.
  Stack:
    Error: Expected 'foo' to equal 'fasdf'.
        at Object.<anonymous> (/Users/Ben/workspace/reactionary/test-test.js:3:19)

1 spec, 1 failure
Finished in 0.011 seconds


Failures:
1) gulp jasmine runs each test only once
1.1) Expected 'foo' to equal 'fasdf'.

1 spec, 1 failure
Finished in 0 seconds
[09:46:37] 'test' errored after 43 ms
[09:46:37] Error in plugin 'gulp-jasmine'
Message:
    Tests failed
```

As you can see, the output for the entire suite prints twice.

Thanks in advance for looking into this. I'm happy to keep using jasmine 2.4.1, but wanted to bring this to your attention.

**Comment by jbblanchet** [comment]
Added a fix in #87. Let me know if it solves your problem. I'll also add an issue to their repo about breaking changes in patch versions.

**Comment by benchristel** [comment]
Yes, that fixed it. Thanks!

**Comment by jbblanchet** [comment]
Solved in 2.4.2. Thanks for reporting.
